### DISCUSSION TITLE
希望能增加其他类型的鼠标手势

### DISCUSSION THREAD

**Discussion by tangjyan** [shuax/MouseInc.Settings#254] [Ideas]
比如说我按住右键，按左键。或者按住右键滚中键这也是一种快速鼠标手势，但是目前是不支持的。希望可以支持这种快速的鼠标手势类型。。

**Comment by shuax** [comment]
现在有WheelSwitchUp手势

**Reply by tangjyan** [reply]
我后来看到了，这个是有的。但是很希望有按住右键，点击左键的手势。将会很方便。

**Reply by modyson** [reply]
我是感觉，现在一个右键的组合，已经够用了

**Comment by starryspace** [comment]
附议，我也需要“按住右键按左键”这个手势，看到其他帖子里也有人提到这个功能，但没看到作者答应会添加这个功能，我也想发一下声，让作者重视重视这个功能。
虽然软件可以画的“鼠标手势”很多，但是复杂的手势使用起不容易，所有手势按照难度从低到高可以分为几下几类：

1. 按键：滚轮上、滚轮下、鼠标左键
2. 单步：上、下、左、右
3. 单步斜向：斜左上、斜右上、斜左下、斜右下
4. ……

简单的手势不多，缺少“鼠标左键”这个，真是有些遗憾呢。

**Comment by 6Tom** [comment]
这确实帅得很，我也想要，不过大佬说过以下这种，所以可能没有类似这种开发计划
![2022-08-29_182924](https://user-images.githubusercontent.com/88385016/187182073-bd59fba7-5555-4fe3-adb3-649703cd67f9.jpg)

**Comment by ysinx** [comment]
附议, 我也需要"按住右键，点击左键", 之前用 WGestures,  把刷新设置成这个, orz orz orz
